Source/WebKit/UIProcess/Automation/protocol/BidiScript.json

```
@@ -0,0 +1,174 @@
+    "exposedAs": "script",
+    "description": "The script module contains commands and events relating to script realms and execution.",
+    "condition": "ENABLE(WEBDRIVER_BIDI)",
+    "types": [
+        {
+            "id": "EvaluateResultType",
+            "type": "string",
+            "enum": [ "success", "exception" ]
+        },
+        {
+            "id": "EvaluateResultOwnership",
+            "type": "string",
+            "enum": [ "root", "none" ]
+        },
+        {
+            "id": "ExceptionDetails",
+            "type": "object",
+            "properties": [
+                { "name": "lineNumber", "type": "number"},
+                { "name": "columnNumber", "type": "number"},
+                { "name": "text", "type": "string" },
+                { "name": "exception", "$ref": "BidiScript.RemoteValue" },
+                { "name": "stackTrace", "$ref": "BidiScript.StackTrace" }
+            ]
+        },
+        {
+            "id": "Handle",
+            "type": "string"
+        },
+        {
+            "id": "InternalId",
+            "type": "string"
+        },
+        {
+            "id": "LocalValue",
+            "description": "Represents a value which can be deserialized into ECMAScript. This includes both primitive and non-primitive values as well as remote references and channels.",
+            "type": "object",
+            "properties": [
+                { "name": "type", "$ref": "BidiScript.LocalValueType" },
+                { "name": "value", "type": "any" }
+            ]
+        },
+        {
+            "id": "LocalValueType",
+            "type": "string",
+            "enum": [
+                "array",
+                "date",
+                "map",
+                "object",
+                "set"
+            ]
+        },
+        {
+            "id": "Realm",
+            "type": "string",
+            "description": "Unique identifier for a script realm. Defined at <https://tc39.es/ecma262/#realm>"
+        },
+        {
+            "id": "RemoteValue",
+            "type": "object",
+            "properties": [
+                { "name": "type", "$ref": "BidiScript.RemoteValueType" },
+                { "name": "value", "type": "any", "optional": true },
+                { "name": "handle", "$ref": "BidiScript.Handle", "optional": true, "description": "Represents a reference to a remote Node. Applicable if `type` property is `node`, `nodelist`, `htmlcollection`." },
+                { "name": "internalId", "$ref": "BidiScript.InternalId", "optional": true, "description": "Represents a reference to a remote script value. If both `handle` and `internalId` are specified, `internalId` takes precedence." }
+            ]
+        },
+        {
+            "id": "RemoteValueType",
+            "type": "string",
+            "enum": [
+                "array",
+                "arraybuffer",
+                "date",
+                "error",
+                "function",
```

Nit: missing `"generator"`?
https://w3c.github.io/webdriver-bidi/#type-script-RemoteValue

```suggestion
                "function",
                "generator",
```